tests/integration_tests/Test/PM/Integration/_data/rest-app/module.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Export generator="Cache" version="25">
+  <Document name="rest-demo.ZPM">
+    <Module>
+      <Name>rest-demo</Name>
+      <Version>1.0.0</Version>
+      <Description>This is a demo of a flask application</Description>
+      <Keywords>flask</Keywords>
+      <Author>
+        <Person>Shuheng Liu</Person>
+        <Organization>InterSystems</Organization>
+        <CopyrightDate>2024</CopyrightDate>
+        <License>MIT</License>
+        <Notes>notes</Notes>
+      </Author>
+      <Packaging>module</Packaging>
+      <SourcesRoot>src</SourcesRoot>
+
+      <WebApplication 
+        Name="/restdemo"
+        Path="/src"
+        Recurse="1"
+        Directory="{$cspdir}/restdemo" 
+        MatchRoles=":{$dbrole}"
+        PasswordAuthEnabled="1"
```

Comment:
Wouldn't this fail? We only have AutheEnabled and provide it values. I would recommend adding a few more options for evaluating each of the $$$Authe* macros. e.g. ${AutheCache}, ${AutheDelegated} that get mapped to the corresponding macro before evaluating

Reply:
Just fixed it by setting `"AutheEnabled` dynamically based on `PasswordAuthEnabled`, `UnauthenticatedEnabled`, etc.